CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 # Change Log
 All notable changes to the MVS-Explorer will be documented in this file.
 
+## <1.0.19>
```

Comment:
1.21 now

Reply:
the last release had tag 1.0.18, so the new one needs to be 1.0.19